transport/http/outbound.go

```diff
@@ -301,26 +300,29 @@ func (o *Outbound) call(ctx context.Context, treq *transport.Request) (*transpor
 
 	hreq, err := o.createRequest(treq)
 	if err != nil {
+		// TODO: Find a way to emit this error to span tag
```

Comment:
What's the need of this comment? Do you have plans to do anything with it?
Besides, if no request was created, how new span may exist?

Reply:
We don't have a plan so far but we want the err to be captured as well eventually. Maybe remove the todo is fine.